controllers/remote/cluster_test.go

```diff
@@ -53,7 +55,7 @@ var (
 	validKubeConfig = `
 clusters:
 - cluster:
-    server: https://test-cluster-api:6443
+    server: https://kubernetes.io
```

Comment:
I'm curious - do the tests fail without this change?

Reply:
When I ran them locally I got this error message when I was creating a the client -
`Get https://test-cluster-api:6443/api?timeout=32s: dial tcp: lookup test-cluster-api: no such host`

Reply:
Ah, it's because the dynamic mapper is trying to talk to the server. You need to create the mapper with an option, `apiutil.WithLazyDiscovery`.

Reply:
Worked! Thank you!